recipe/meta.yaml

```
@@ -1,16 +1,17 @@
-{% set version = "1.3.1" %}
+{% set name = "qtawesome" %}
+{% set version = "1.4.0" %}
 
 package:
-  name: qtawesome
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/q/qtawesome/QtAwesome-{{ version }}.tar.gz
-  sha256: 075b2c9ee01cbaf5e3a4bebed0e5529ee8605981355f21dea051b15c1b869e1b
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qtawesome-{{ version }}.tar.gz
+  sha256: 783e414d1317f3e978bf67ea8e8a1b1498bad9dbd305dec814027e3b50521be6
 
 build:
   number: 0
-  skip: true  # [py<37 or ppc64le or s390x]
+  skip: true  # [py<37]
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir --no-build-isolation -vvv
   entry_points:
     - qta-browser=qtawesome.icon_browser:run
```

```suggestion
    - qta-browser=qtawesome.icon_browser:run
    - qta-install-fonts-all-users=qtawesome:install_bundled_fonts_system_wide
```
see https://github.com/spyder-ide/qtawesome/blob/v1.4.0/setup.py#L56C14-L56C85

Can you test the command if it doesn't require a browser?

---

`qta-browser --help` opens a GUI application